packages/sdks/typescript/src/index.ts

```diff
@@ -74,6 +97,38 @@ export class Latitude {
     this.source = __internal.source
   }
 
+  async get(
+    path: string,
+    {
+      projectId,
+      versionUuid,
+    }: {
+      projectId?: number
+      versionUuid?: string
+    } = {},
+  ) {
+    projectId = projectId ?? this.projectId
+    if (!projectId) {
+      throw new Error('Project ID is required')
+    }
+
+    const response = await this.request({
+      method: 'GET',
+      handler: HandlerType.GetDocument,
+      params: { projectId, versionUuid, path },
+    })
+
+    if (!response.ok) {
+      throw new LatitudeApiError({
+        status: response.status,
+        statusText: response.statusText,
+        serverResponse: await response.text(),
+      })
+    }
+
+    return (await response.json()) as DocumentVersion & { config: Config }
```

Comment:
I realize we're not handling `.json()`. This can fail no?

Reply:
yes it can